build/common/installer/scripts/tomlparser-prom-agent-config.rb

```diff
@@ -14,7 +14,7 @@
 @promFbitMemBufLimit = 200
 
 @waittime_port_25226 = 45
-@waittime_port_25228 = 60
```

Comment:
will 90s sufficient or it must be 120s

Reply:
So, may be 90 is sufficient. Adding a higher value so we can see what is the max needed from telemetry and then make a call to reduce it accordingly